build.sbt

```diff
@@ -45,7 +45,7 @@ lazy val commonDependencies = Seq(
 lazy val buildSettings = Seq(
   organization := "com.snowplowanalytics",
   name := "snowplow-stream-collector",
-  version := "2.0.1",
+  version := "2.0.2-rc1",
```

Comment:
Maybe the tag can make it clearer that this is not part of the normal release process and this will not be merged to `master` but just used from the branch? On that note, maybe it shouldn't be on a `fix/*` branch but rather some sort of `emergency/kinesis-down` branch?

Reply:
Totally agree, would be good to have such convention. I should use hotfix at least, as in gitflow workflow, sorry.